zio-http/jvm/src/test/scala/zio/http/endpoint/openapi/OpenAPIGenSpec.scala

```diff
@@ -165,7 +165,7 @@ object OpenAPIGenSpec extends ZIOSpecDefault {
       (GET / "static" / int("id") / uuid("uuid") ?? Doc.p("user id") / string("name")) ?? Doc.p("get path"),
     )
       .in[SimpleInputBody](Doc.p("input body"))
-      .out[SimpleOutputBody](Doc.p("output body"))
+      .out[SimpleOutputBody](mediaType = MediaType.application.json, Doc.p("output body"))
```

Comment:
Why?

Reply:
Probably to test this: https://github.com/zio/zio-http/pull/3044#discussion_r1726522056

Reply:
Yes, this is to test the exact issue described in #3044